### ISSUE TITLE
Add friend world download?

### ISSUE THREAD

**Issue by ChickenCMB** [bedrock-tool/bedrocktool#111]
![Screenshot_3](https://github.com/bedrock-tool/bedrocktool/assets/145242044/91e57a1c-da35-4646-8e26-7da9f522bab8)
 Added feature to download worlds from friends?
can it be?

**Comment by BedrockToolLoverAchieved** [comment]
I don't think this would be feasible due to the legendary BedrockTool being command-prompt-based.

**Comment by olebeck** [comment]
this requires an xbox api, this feature would work similar to realms, dont have the api implemented yet

**Comment by ChickenCMB** [comment]
> this requires an xbox api, this feature would work similar to realms, dont have the api implemented yet

Hope it releases soon, waiting... By the way your tool are good
